ui/src/app/components/source-selection/source-selection.component.html

```diff
@@ -1,12 +1,12 @@
 <div *ngIf="!isDatabaseLoading">
   <div class="prerequisites">
     <mat-card>
-      <mat-card-title>Prerequisites</mat-card-title>
-      <mat-card-subtitle
+      <mat-card-title class="primary-header">Prerequisites</mat-card-title>
+      <mat-card-subtitle class="body-font-size"
         >Before we begin, please ensure you have done the following:</mat-card-subtitle
```

Comment:
The Typography should be black inside the Prerequisites as suggested in the feedback doc: https://docs.google.com/document/d/1dfDHYSvSiQyyKrfBcuGLUacY6BbPtyXOBcC9y769ZZs/edit

Reply:
Updated